drivers/modem/gsm_ppp.c

```diff
@@ -523,9 +523,12 @@ MODEM_CMD_DEFINE(on_cmd_atcmdinfo_attached)
 
 static const struct modem_cmd read_cops_cmd =
 	MODEM_CMD_ARGS_MAX("+COPS:", on_cmd_atcmdinfo_cops, 1U, 4U, ",");
-
 static const struct modem_cmd check_net_reg_cmd =
+#if defined(CONFIG_MODEM_GSM_SELECT_CEREG)
+	MODEM_CMD("+CEREG: ", on_cmd_net_reg_sts, 2U, ",");
```

Comment:
then you  can do this in one liner by concatenating:
`"+" "CONFIG_MODEM_GSM_STATUS_COMMAND"`